go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/crossplane/provider-aws
 go 1.13
 
 require (
+	github.com/aws-controllers-k8s/runtime v0.1.1
```

Comment:
Oh and you need to regenerate using the latest ACK code-generator that has my fixes that removes this dependency.